src/together/cli/api/finetune.py

```diff
@@ -133,13 +146,17 @@ def create(
         lora_trainable_modules=lora_trainable_modules,
         suffix=suffix,
         wandb_api_key=wandb_api_key,
+        train_on_inputs=train_on_inputs,
     )
 
     model_limits: FinetuneTrainingLimits = client.fine_tuning.get_model_limits(
         model=model
     )
 
     if lora:
+        log_warn_once(
+            "LoRA rank default has been changed from 8 to 64 as the maximum available for each model."
```

Comment:
Does this mean that every model supported by the API has 64 as the maximum possible rank? I think the current wording is confusing if this is not true